vendor/github.com/OpenBazaar/go-ethwallet/wallet/wallet.go

```diff
@@ -788,7 +686,6 @@ func (wallet *EthereumWallet) AssociateTransactionWithOrder(txnCB wi.Transaction
 
 // CheckTxnRcpt check the txn rcpt status
 func (wallet *EthereumWallet) CheckTxnRcpt(hash *common.Hash, data []byte) (*common.Hash, error) {
```

Comment:
Can we remove the `data` arg entirely and instead use `hash` to `wallet.db.Txns().Get(hash)` so we aren't accidentally operating on a different version of `data` from one execution path to another?

Reply:
This func is now internal and called from one place inside the wallet only.